descriptors/ModuleDescriptor-template.json

```diff
@@ -2608,6 +2608,7 @@
       "displayName": "inventory storage module - all permissions",
       "description": "Entire set of permissions needed to use the inventory storage module",
       "subPermissions": [
+        "inventory-storage.related-instances.collection.get",
```

Comment:
This is only a single permission. I thought that multiple new permissions were added for the new endpoints?

Reply:
Yeah, I for some reason thought you were asking to only have the one permission placed here.  Just added them all.  also the related-instance-type endpoint permissions didn't seem to be here either, so added those as well.

Reply:
Thanks